File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TaggedAssetBase.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id The ID of the Asset being tagged.
 */
data class TaggedAssetBase (

        @get:NotNull 
        @JsonProperty("id") val id: Long
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TaggedAssetBase.scala

package org.openapitools.client.model


case class TaggedAssetBase (
    /* The ID of the Asset being tagged. */
    _id: Long
)
object TaggedAssetBase {
    def toStringBody(var_id: Object) =
        s"""
        | {
        | "id":$var_id
        | }
        """.stripMargin
}
